composer.json

```diff
@@ -21,5 +25,16 @@
         "psr-4": {
             "ColoredCow\\LaravelMobileAPI\\Test\\": "tests/"
         }
+    },
+    "extra": {
+        "laravel": {
+            "providers": [
+                "ColoredCow\\LaravelMobileAPI\\MobileAPIServiceProvider",
+                "ColoredCow\\LaravelMobileAPI\\MobileAuthAPIServiceProvider"
```

Comment:
We can remove this service provider. All the registration and updates can happen from a single service provider only for this package.

Reply:
@pankaj-ag Please add your inputs here. Passport requires the `Passport::routes` to be included from the auth service provider only.

Reply:
@bohraji @pankaj-ag I have removed the `ColoredCow\\LaravelMobileAPI\\MobileAuthAPIServiceProvider` and register route in the `"ColoredCow\\LaravelMobileAPI\\MobileAPIServiceProvider"` 
after testing the routes.